lighthouse-core/audits/installable-manifest.js

```diff
@@ -7,6 +7,20 @@
 
 const MultiCheckAudit = require('./multi-check-audit.js');
 const ManifestValues = require('../computed/manifest-values.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the desktop installability of a website. This descriptive title is shown to users when a webapp is installable. */
```

Comment:
definitely not desktop only. Maybe something about if the web site can be installed as a standalone app?